analyses/chromothripsis/02-run-shatterseek-and-classify-confidence.R

```diff
@@ -96,7 +109,6 @@ mergeCNsegments <- function(cnv_df) {
   # does not match the previous row (starting at second row)
   cnv_df$index <- 0
   index_counter <- 1
-  if (nrow(cnv_df) >=2) {
```

Comment:
This was Krutika's edit for debugging - I moved down below (line 163 in the updated file)